src/models/ratings/tasks/rating.task.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { Cron } from '@nestjs/schedule';
```

Comment:
TIL about node-cron. wow.

Reply:
yeah it's a nifty package and pretty easy to setup in nest.